scene/2d/gpu_particles_2d.cpp

```diff
@@ -588,6 +593,9 @@ GPUParticles2D::GPUParticles2D() {
 	set_speed_scale(1);
 	set_fixed_fps(30);
 	set_collision_base_size(collision_base_size);
+#ifdef TOOLS_ENABLED
+	set_show_visibility_rect(false);
```

Comment:
`show_visibility_rect = false` would be enough. Other properties are set via methods, because they do something with RenderingServer.

Reply:
Thanks, fixed